From f3c7b8f6b90f52efe86fdc8bcb0c26c3734521ab Mon Sep 17 00:00:00 2001
From: andip71 <andreasp@gmx.de>
Date: Sun, 22 Feb 2015 10:53:27 +0100
Subject: [PATCH] b

---
 init.rc | 11 +++++++++--
 1 file changed, 9 insertions(+), 2 deletions(-)

diff --git a/init.rc b/init.rc
index 32d33d8..9f5807f 100755
--- a/init.rc
+++ b/init.rc
@@ -349,9 +349,9 @@ on boot
     write /proc/sys/vm/overcommit_memory 1
     write /proc/sys/vm/min_free_order_shift 4
     chown root system /sys/module/lowmemorykiller/parameters/adj
-    chmod 0220 /sys/module/lowmemorykiller/parameters/adj
+    chmod 0660 /sys/module/lowmemorykiller/parameters/adj
     chown root system /sys/module/lowmemorykiller/parameters/minfree
-    chmod 0220 /sys/module/lowmemorykiller/parameters/minfree
+    chmod 0660 /sys/module/lowmemorykiller/parameters/minfree
 
     # Tweak background writeout
     write /proc/sys/vm/dirty_expire_centisecs 200
@@ -655,3 +655,10 @@ service pre-recovery /system/bin/uncrypt
     class main
     disabled
     oneshot
+
+# Call Boeffla-Kernel init script
+service boeffla-init /sbin/boeffla-init.sh
+    class main
+    user root
+    group root
+    oneshot
-- 
1.9.1

